spyderlib/widgets/sourcecode/syntaxhighlighters.py

```diff
@@ -414,6 +414,20 @@ def highlightBlock(self, text):
                     
             match = self.PROG.search(text, match.end())
  
+        # Make blank space less apparent by setting the foreground alpha.
+        # This only has an effect when "Show blank space" is turned on.
+        re_blank = re.compile("\s+")
+        match = re_blank.search(text, offset)
+        while match:
+            start, end = match.span()
+            start = max([0, start+offset])
+            end = max([0, end+offset])
+            format = self.format(start)
+            color_foreground = format.foreground().color()
+            color_foreground.setAlpha(80)

```

Comment:
Perhaps since you are hard coding the alpha value, it would be better to have it in a more "accessible/set-able" place? @Nodd suggestions on this? 

maybe 

```python
            color_foreground = format.foreground().color()
            color_foreground.setAlpha(self.SPACE_TAB_ALPHA)`

And `SPACE_TAB_ALPHA` can be also a class variable?